### DISCUSSION TITLE
How is user's data stored

### DISCUSSION THREAD

**Discussion by sveetya** [ghostfolio/ghostfolio#903] [Q&A] [answered]
How is the user's data stored on the server, is it encrypted? If yes, how?

**Comment by dtslvr** [comment, marked as answer]
Hello @Marilok

The data is currently not encrypted. Users of the SaaS can use [Ghostfol.io](https://ghostfol.io) anonymously and self-hosters could encrypt the data beyond the application level (database, hard drive).

Does that answer your question?
